robottelo/install/base.py

```diff
@@ -0,0 +1,152 @@
+# -*- encoding: utf-8 -*-
+"""Generic base class for install commands."""
+import logging
+
+from robottelo import ssh
+from robottelo.config import conf
+
+
+class InstallError(Exception):
+    """Indicates that a install command could not be run."""
+
+
+class Base(object):
+    """
+    @param command_base: base install command.
+    """
+    command_base = None  # each inherited instance should define this
+    command_requires_auth = False  # True when command requires auth user
+    command_service = None
+
+    logger = logging.getLogger('robottelo')
+
+    @classmethod
+    def reset(cls, options=None):
+        """
+        dummy
+        """
+
+        if options is None:
+            options = {}
+
+        result = cls.execute(cls._construct_command(options))
+        return result
+
+    @classmethod
+    def execute(cls, command, user=None, password=None, timeout=None):
```

Comment:
Repeating the same code from Base CLI. Why not inherit and leverage the already implemented methods? We must follow DRY to avoid future maintenance issues.